WORKSPACE

```diff
@@ -17,9 +17,23 @@ pip_repositories()
 
 load("@io_bazel_rules_python//python:pip.bzl", "pip_import")
 pip_import(
-   name = "hello_world_python_deps",
-   requirements = "//pores4thought/hello_world/python:requirements.txt",
+   name = "python_deps",
+   requirements = "//third_party:requirements.txt",
 )
 
-load("@hello_world_python_deps//:requirements.bzl", "pip_install")
+load("@python_deps//:requirements.bzl", "pip_install")
```

Comment:
Moved python dependency target from hello world to more general third_party location